doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -2,55 +2,44 @@
-* Programmatic autoscaling hints.
+**max_workers[default_value=2, min_value=0]**: Indicates the maximum number of workers the cluster will have at any given time. This number
+excludes the head node where there is always exactly one.
 
-Take a look at :ref:`the cluster reference <cluster-config>` to learn more
-about configuring the autoscaler.
+.. note::
+  When this value changes and drops below the number of running workers, the autoscaler will remove nodes to satisfy the new constraint even
+  if the node is running some workload. Workers that are the least recently used will be removed first.
 
+**upscaling_speed[default_value=1.0, min_value=0.0]**: The number of nodes allowed to be pending as a multiple of the current number of nodes.
+For example, if set to 1.0, the cluster can grow in size by at most 100% at any time, so if the cluster currently has 20 nodes, at most 20 pending
+launches are allowed. The mininum number of pending launches is 5 regardless of this setting.
 
-How does it work?
-^^^^^^^^^^^^^^^^^
+**idle_timeout_minutes[default_value=5, min_value=0]**: The number of minutes that need to pass before an idle worker node is removed by the
+autoscaler. Worker nodes are idle when there are no active tasks or actors running. This parameter does not affect the head node.
 
-The Ray Cluster Launcher will automatically enable a load-based autoscaler. The
-autoscaler resource demand scheduler will look at the pending tasks, actors,
-and placement groups resource demands from the cluster, and try to add the
-minimum list of nodes that can fulfill these demands. Autoscaler uses a simple 
-binpacking algorithm to binpack the user demands into
-the available cluster resources. The remaining unfulfilled demands are placed
-on the smallest list of nodes that satisfies the demand while maximizing
-utilization (starting from the smallest node).
+.. note::
+  If the Ray object store is used, and a worker node still holds objects (including spilled objects on disk), it won't be removed.
 
-**Downscaling**: When worker nodes are
-idle (without active Tasks or Actors running on it) 
-for more than :ref:`idle_timeout_minutes
-<cluster-configuration-idle-timeout-minutes>`, they are subject to
-removal from the cluster. But there are two important additional conditions
-to note: 
+**available_node_types.<node_type_name>.min_workers[default_value=0, min_value=0]**: The minimum number of worker nodes of a given type to launch. If this number
+is set to greater than zero, the autoscaler will maintain the number of nodes regardless of utilization. The sum of the min worker of all the node types
+must be less than or equal to the max_workers for the cluster.
 
-* The head node is never removed unless the cluster is torn down.
-* If the Ray Object Store is used, and a Worker node still holds objects (including spilled objects on disk), it won't be removed.
-
-
-
-**Here is "A Glimpse into the Ray Autoscaler" and how to debug/monitor your cluster:**
```

Comment:
Should we chuck this video in a `..tip` somewhere so that we don't lose it?

Reply:
is this the right page for that video? 

seems like an overview of autoscaler a not about configuring it

Reply:
I'd be fine with moving it to a different page, just don't want it to disappear from the docs entirely

Reply:
Definitely should keep the video somewhere.

Reply:
https://github.com/ray-project/ray/issues/27571

Reply:
IMO this page is as good as any for linking to this video. Users who want to understand more about the autoscaler on VM stacks are sure to visit this page. We don't have to add a lot of explanation -- just a link with a brief description is fine.

Reply:
Do we have autoscaler in cluster overview or key concepts?

Reply:
We currently have three sections dedicated to autoscaling:
* [Autoscaling in Key Concepts](https://ray--27510.org.readthedocs.build/en/27510/cluster/cluster_under_construction/key-concepts.html#autoscaler) (in PR, almost merged)
* [Autoscaling](https://docs.ray.io/en/master/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/autoscaling.html) (for Kubernetes users)
* This page

The linked video discusses how to configure the autoscaler using the cluster YAML file. I think the best place to link to the video is from this page, since it 1) is VM-specific and 2) discusses the same topic as the video (the autoscaler knobs).

WDYT?